data/mods/ssb/conditions.ts

```diff
@@ -197,6 +197,30 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			this.add(`c:|${getName('Kennedy')}|LOOOOOOL ffs`);
 		},
 	},
+	kolohe: {
```

Comment:
Rename all traces of Kolohe to be their PS name Kolochu

Reply:
PS names MUST be used for Staff Bros. No exceptions unless you have that PS username